lib/@types/Identity.ts

```diff
@@ -32,3 +32,9 @@ export type Identity = {
   type: IdentityTypes;
   value: string;
 };
+
+export type InMemoryTemporaryIdentity = {
+  value: string;
+  type: IdentityTypes;
+  expiresAt: number;
+};
```

Comment:
This should stay outside of `lib`, because it's related to how the client keeps the temporal identity in memory (through `useState`).

Reply:
OK fixed d5c42fa